adobecom/milo
# MWPW-138808 Apply leaner fix for merch card prices (web components)

This PR reverts changes made by https://github.com/adobecom/milo/pull/1478 and https://github.com/adobecom/milo/pull/1474 and fixes root cause of the issue with custom web components in Safari browser.

Resolves: [MWPW-138808](https://jira.corp.adobe.com/browse/MWPW-138808)

Before: https://main--milo--adobecom.hlx.page/drafts/vlassenko/share-pdf-online?martech=off
After: https://mwpw-138808--milo--adobecom.hlx.page/drafts/vlassenko/share-pdf-online?martech=off

**QA notes**

Pages to check:
- https://mwpw-138808--milo--adobecom.hlx.page/drafts/vlassenko/share-pdf-online?martech=off (Merch cards in the "Choose the Acrobat plan that works for you" section)
- https://stage--homepage--adobecom.hlx.page/drafts/seanchoi/modal-test?milolibs=mwpw-138808 ("Save now" button on the "Early Black Friday deal" card)
- https://mwpw-138808--milo--adobecom.hlx.page/drafts/ilyas/merch (all checkout link and inline price components)

On the test pages, each price placeholder should be rendered as `<span is="inline-price">` element, each checkout CTA - as `<a is="checkout-link">` element.
